securing_apps/topics/saml/mod-auth-mellon.adoc

```diff
@@ -68,9 +70,7 @@ Apache configuration directives typically follow a hierarchical tree structure i
 
 This example has just one protected location: \https://$sp_host/private.
 
-To configure the Mellon service provider, complete the following steps:
-
-. Create the file /etc/httpd/conf.d/mellon.conf with this content:
+To configure the Mellon service provider, create the file /etc/httpd/conf.d/mellon.conf with this content:
```

Comment:
Mellon is actually an Apache module ...

Reply:
@pedroigor What do I need to change here.